mu/interface.py

```diff
@@ -755,6 +760,30 @@ def __init__(self, port, theme='day', parent=None):
             raise IOError("Cannot connect to device on port {}".format(port))
         self.set_theme(theme)
 
+    def repl_paste(self):
+        """
+        Grabs clipboard contents then sends down the serial port
+        """
+        clipboard = QApplication.clipboard()
+        self.serial.write(bytes(clipboard.text(), 'utf8'))
+
+    def repl_context_menu(self):
+        """"
+        Creates custom context menu with just copy and paste
+        """
+        menu = QMenu(self)
+        menu.addAction("Copy", self.copy)
+        menu.addAction("Paste", self.repl_paste)
+        menu.exec_(QCursor.pos())
+
+    def cursor_to_end(self):
```

Comment:
If it gets called when the user clicks or changes the cursor, would this method move the cursor to the end of the selected line for all lines, would that be correct?
Does that affect text selection in case the user might want to copy some data from the repl?